elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -0,0 +1,64 @@
+{{#question}}
+    <div class="row">
+        <div class="column {{dropzone_layout}}">
+            <ul id="{{answer_name}}-options" name="{{answer_name}}" class="connectedSortable">
+                <li class='ui-sortable-handle info info-fixed'>{{header-left-column}}</li>
+                    {{#options}}
+                        <li string="{{.}}" title="{{.}}">{{.}}</li>
+                    {{/options}}
+                </ul>
+        </div>
+            <div class="column {{dropzone_layout}}">
+                <ul id="{{answer_name}}-dropzone" name="{{answer_name}}" class="connectedSortable dropzone">
+                    <li class="ui-sortable-handle info info-fixed">{{header-right-column}}</li>
+                    {{#submission_dict}}
+                        <li string="{{text}}" title="{{text}}" style='margin-left: {{indent}}px;'>{{text}}</li>
+                    {{/submission_dict}}
+                </ul>
+            </div>
+        </div>
+    <input id="{{answer_name}}-input" type="hidden" name="{{answer_name}}-input" value=""/>
+
+{{/question}}
+
+{{#submission}}
+    {{^parse-error}}
+        <div style="margin-bottom: 5px;"><strong>Your answer: </strong></div>
+        {{#student_submission}}
+        	{{#render_as_code}}
+        		<code class="user-output"> {{text}} </code> 
+        	{{/render_as_code}}
+        	{{^render_as_code}}
+        		<p> {{text}}</p>
+        	{{/render_as_code}}
```

Comment:
It kind of seems weird to me that this has to be an option. Can't it just render the submission using the exact same formatting that the question-writer used when they wrote the question? It seems like we should be able to just do the right thing automatically instead of the question writer needing to worry about setting this option correctly.

Reply:
`render_as_code` is an internal parameter that automatically detects whether the answer text contains LaTeX text; it is automatically set by the code. I only added the seemingly redundant parameter here because of mustache limitations.

Reply:
I still don't see why wrap the content in the `code` element. If the question writer wanted to have it displayed as code, wouldn't the way they originally wrote the question take care of that?

Reply:
the styling was suggested by Andrew to try and keep the styling of `pl-order-blocks` consistent with other elements such as `pl-dropdown`, who also wraps answers within code  blocks

![圖片](https://user-images.githubusercontent.com/11191061/97912385-982a1680-1d01-11eb-829d-1c5f40bdfb4b.png)

I could change it, because not all elements follow this styling

Reply:
I'm actually not sure why `pl-dropdown` uses `<code>` styling for the answers. This styling _is_ used by `pl-string-input` because it's trying to show the specific characters that the student typed, but for `pl-dropdown` and `pl-order-blocks` the strings we are displaying are not entered by the student and so I agree with @SethPoulsen that we should always display in a normal styling (that is, no `<code>`).

Reply:
resolved in latest commit